crates/hyperqueue/src/client/output/mod.rs

```diff
@@ -1,5 +1,5 @@
 pub mod cli;
-mod common;
+pub(crate) mod common;
```

Comment:
You can also just export a single function like this:
```rust
pub use common::resolve_task_paths;
```
and keep `common` private.